package.json

```diff
@@ -24,6 +24,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.3.0",
     "redux": "^4.0.4",
+    "redux-persist": "^6.0.0",
```

Comment:
We're adding this, but not using it as far as I can tell?

Reply:
I have removed it.